NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -26,8 +26,19 @@ public void BroadcastItemAdd(Pickupable pickupable, Transform containerTransform
         {
             NitroxId itemId = NitroxEntity.GetId(pickupable.gameObject);
             byte[] bytes = SerializationHelper.GetBytes(pickupable.gameObject);
+            NitroxId ownerId = GetOwner(containerTransform);
 
-            ItemData itemData = new ItemData(GetOwner(containerTransform), itemId, bytes);
+            ItemData itemData;
+            Plantable plant = pickupable.GetComponent<Plantable>();
+            if (plant && plant.currentPlanter)
+            {
+                // special case: we want to remember the time when the plant was added, so we can simulate growth
+                itemData = new PlantableItemData(ownerId, itemId, bytes, DayNightCycle.main.timePassedAsDouble);
```

Comment:
Is timePassedAsDouble not serialized in bytes?  Is there a way to add it?

Reply:
`class Plantable` does contain `public float plantAge`, but storing that won't help without any reference time.